sys/net/gnrc/application_layer/uhcpc/gnrc_uhcpc.c

```diff
@@ -87,6 +90,15 @@ void uhcp_handle_prefix(uint8_t *prefix, uint8_t prefix_len, uint16_t lifetime,
 
     gnrc_netapi_set(gnrc_wireless_interface, NETOPT_IPV6_ADDR, (64 << 8),
```

Comment:
Off-topic: I took the 64 from the original implentation (see 861035f22d). Any idea, why `prefix_len` wasn't used here?